package.json

```diff
@@ -17,30 +17,41 @@
   },
   "homepage": "https://github.com/digitalbazaar/bedrock-vc-verifier",
   "dependencies": {
-    "axios": "^0.19.2",
-    "bedrock-did-context": "^1.0.0",
-    "bedrock-veres-one-context": "^10.0.0",
+    "@digitalbazaar/ed25519-signature-2018": "^2.0.1",
+    "@digitalbazaar/ed25519-signature-2020": "^2.2.0",
+    "@digitalbazaar/ed25519-verification-key-2018": "^3.1.1",
+    "@digitalbazaar/http-client": "^1.1.0",
+    "@digitalbazaar/vc": "^1.0.0",
+    "bedrock-did-context": "^2.0.0",
+    "bedrock-security-context": "^4.1.0",
+    "bedrock-veres-one-context": "^11.0.0",
     "body-parser": "^1.19.0",
     "cors": "^2.8.5",
-    "did-io": "^0.8.2",
-    "did-method-key": "^0.6.1",
-    "did-veres-one": "^12.1.1",
-    "serialize-error": "^6.0.0",
-    "vc-js": "^0.6.1",
-    "vc-revocation-list": "^1.0.0"
+    "crypto-ld": "^6.0.0",
+    "did-context": "^3.0.1",
+    "serialize-error": "^8.1.0",
+    "vc-revocation-list": "^1.0.0",
```

Comment:
This dependencies should be updated: https://github.com/digitalbazaar/vc-revocation-list/blob/main/CHANGELOG.md#300---2021-05-24

Please include in the the breaking changed section here:

**BREAKING**: Revocation list credentials must have the same issuer value as the credential to be revoked.